.github/workflows/wheels.yml

```diff
@@ -6,6 +6,15 @@
 # in the get_commit_message job. Be sure to include [wheel build] in your commit
 # message to trigger the build. All files related to wheel building are located
 # at tools/wheels/
+# Alternatively, you can add labels to the pull request in order to trigger wheel
+# builds.
+# The labels that trigger builds are: 
+# 36 - Build(for changes to the building process, 
+# 03 - Maintenance(for dependency upgrades), and 
+# 14 - Release(ensure wheels build before release)
+# Note: Adding a label will not run the wheel builders on an commit made before the label was added.
+# You will need to close/re-open the pull request or add another commit to trigger the wheel builders after a
```

Comment:
Apologies if this was considered and discounted, but you could solve that by adding below under `on:`:
```yaml
    pull_request:
      types:
        - labeled
        - synchronize
```

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Would that run wheel builds only on the specific labels listed in the comment?

Reply:
Unfortunately, I think the problem with this approach is that the completed runs only show up as a check on the commit itself but not under the ``Checks`` tab for a pull request.

Reply:
No worries, I wasn't sure after you said that so I only just now verified that it works as expected. I know my experience with GHA is that "as expected" doesn't mean reality.